index.html

```diff
@@ -1008,6 +1029,14 @@ <h2>Sharing theme</h2>
             return defaultValue;
           }
         },
+        logEvent(eventType, eventProperties) {
+          try {
+            amplitude.getInstance().logEvent(eventType, eventProperties);
+          } catch (error) {
```

Comment:
A more robust approach would be to queue them until it's available but probably not necessary given these won't happen on page load

Reply:
We might want an initial event, but the first event we have right now is `level_start` and that doesn't happen until the first guess is made.

Reply:
And in my testing amplitude actually loaded very quickly. I wasn't even able to trigger this catch with `Slow 3G` simulation on.